src/client/components/Home/RegisterButton.style.css

```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
```

Comment:
We need font appropriate font size for h2 and p

Reply:
they didn't mentioned any font-size, any suggestions??